From e9947d8a4041b7f194b988aed6f15bd59ea8150b Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Mon, 17 May 2021 19:46:42 +0200
Subject: [PATCH] Disabled resync throtteling.

We currently don't have info about disk I/O in flight, so
this feature is disabled at the moment. We can enable it
once disk stat is implemented. This patch is for DRBD 9.0.29
---
 drbd/drbd_receiver.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drbd/drbd_receiver.c b/drbd/drbd_receiver.c
index 511145f..c8cf9dc 100644
--- a/drbd/drbd_receiver.c
+++ b/drbd/drbd_receiver.c
@@ -3385,8 +3385,8 @@ bool drbd_rs_c_min_rate_throttle(struct drbd_peer_device *peer_device)
 	if (c_min_rate == 0)
 		return false;
 
-	curr_events = (int)part_stat_read_accum(disk->part0, sectors)
-		- atomic_read(&device->rs_sect_ev);
+/* TODO: WinDRBD: resync throtteling disabled for now.  */
+	curr_events = 0;
 
 	if (atomic_read(&device->ap_actlog_cnt) || curr_events - peer_device->rs_last_events > 64) {
 		ULONG_PTR rs_left;
-- 
2.17.0

